mitiq/zne/inference.py

```diff
@@ -1560,36 +1560,49 @@ def extrapolate(
             data of an instantiated Factory object, the bound method
             ".reduce()" should be called instead.
         """
+
         with pm.Model():
             """
-            We assumme that the priors for the model parameters is a uniform
-            distribution with an upper limit 1 and a lower limit 0,
-            except for b with a lower limit -1.
+            We assumme that the priors for the model parameters are normally
+            distributed, while the standard deviation is half normal.
             """
-            a = pm.Uniform('a', 0, 1)
-            b = pm.Uniform('b', -1, 1)
-            c = pm.Uniform('c', 0, 1)
-            eps = pm.Uniform('eps', 0, 0.5)
+            a = pm.Normal('a', 0, 5)
+            b = pm.Normal('b', 0, 5)
+            c = pm.Normal('c', 0, 5)
+            sd = pm.HalfNormal('sd', 10)
 
             pm.Normal(
                 'expval',
                 mu=ExpBayesFactory._exp_ansatz(a, b, c, scale_factors),
-                sd=eps,
+                sd=sd,
                 observed=exp_values,
             )
 
-            trace = pm.sample(target_accept=0.95)
+            trace = pm.sample(target_accept=0.95, return_inferencedata=True)
+
+        def zne_curve(scale_factor: float) -> float:
```

Comment:
🥳  Yeah I think this addresses what was needed to make sure we were avoiding the estimate of a model != model of the estimate